README.md

```diff
@@ -1,5 +1,62 @@
-# AdvancedPlHide
+<!--suppress HtmlDeprecatedAttribute -->
+<div align="center">
+<img alt="AdvancePlHide" src="./.github/banner.png" />
 
-Hide your plugin list by blocking `/plugins` and managing tab completion.
+<!-- Shields -->
+[Document (not finshed)]:https://img.shields.io/badge/-Document-blue.svg?logo=Wikipedia&style=for-the-badge&logoColor=black
+[Discord]:https://img.shields.io/badge/-Discord-5865F2.svg?logo=discord&style=for-the-badge&logoColor=white
+[Spigot]:https://img.shields.io/badge/-SpigotMC-ef9023.svg?logo=Accenture&style=for-the-badge&logoColor=grey
 
-# Documentation soon
+[SpigotRating]:https://img.shields.io/spiget/rating/98645?style=flat-square
+[GitHubStar]:https://img.shields.io/github/stars/BlueTree242/AdvancedPlHide
+[TestedVersion]: https://img.shields.io/spiget/tested-versions/98645?label=Tested%20on&style=flat-square
+[CodeSize]:https://img.shields.io/github/languages/code-size/BlueTree242/AdvancedPlHide
+<!-- Shields -->
+[![Discord]](https://discordsrvutils.xyz/support)
+[![Spigot]](https://spigotmc.org/resources/98645)
+<br><br>[![TestedVersion]](https://spigotmc.org/resources/98645) ![SpigotRating]
+<br> ![GitHubStar] ![CodeSize]
+</div>
+
+## Table of Contents
+* [Introduction](#introduction)
+  * [Reasons](#Reasons)
+  * [Features](#features)
+* [Requirements](#Requirements)
+* [Installation](#Installation)
+
+## Introduction
+
+### Reasons
+
+* This plugin was made to replace any paid plugin hiding
+
+### Features
+
+* Group merging.
+* Hides /pl (/plugin).
```

Comment:
/plugins not /plugin

Reply:
?

Reply:
uk that works as well
test it

Reply:
![image](https://user-images.githubusercontent.com/69328929/147374410-4e68be65-b886-4451-8bb3-767aef9823df.png)

Reply:
DONNNNNNE